mybackend/models/Competition.js

```diff
@@ -1,49 +1,63 @@
 const mongoose = require('mongoose');
+const encrypt = require('mongoose-encryption');
+require('dotenv').config(); // To access environment variables
 
 const competitionSchema = new mongoose.Schema({
   name: String,
   description: String,
   status: { type: String, default: 'Live' },
-  reward: Number,  
-  rewards: {      
+  reward: Number,
+  rewards: {
     feature: Number,
     bug: Number,
     optimization: Number,
     security: Number,
   },
-  points: Number,  // XP points for the competition
-  languages: [String],  // Programming languages for the competition
-  types: [String],  // Types of submissions (Feature, Bug, Optimization, etc.)
-  endDate: Date,    // End date of the competition
-  startDate: Date,  // Start date of the competition
-  image: String,    // Image link for competition
-  websiteLink: String,  // Link to the competition website
-  repositoryLink: String,  // Link to the repository for submissions
-  competitionDetails: String,  // Detailed description of the competition
-  howToGuide: String,  // Guide on how to participate
-  scope: String,  // Scope of the competition
-  judges: {       // Judges for the competition
+  points: Number, // XP points for the competition
```

Comment:
rename this to xpPoints then we don't need the comment. Add a comment at the top to say what the below key value pairs are for.